driver/flambda_backend_args.ml

```diff
@@ -107,38 +107,37 @@ let mk_zero_alloc_check f =
   " Check that annotated functions do not allocate \
    and do not have indirect calls. "^Zero_alloc_annotations.doc
 
-let mk_dcheckmach f =
-  "-dcheckmach", Arg.Unit f, " (undocumented)"
+let mk_dzero_alloc f =
+  "-dzero-alloc", Arg.Unit f, " (undocumented)"
 
-let mk_disable_checkmach f =
-  "-disable-checkmach", Arg.Unit f,
+let mk_disable_zero_alloc_checker f =
+  "-disable-zero-alloc-checker", Arg.Unit f,
```

Comment:
Does the build system ever generate these flags?  If so, it might be better to add the new ones as aliases of the old ones, for now, so we don't end up gating a compiler roll behind a build system roll.

Reply:
We won't have such a dependency, as far as I know.